SECURITY.md

```diff
@@ -20,22 +20,33 @@ TorchServe as much as possible relies on automated tools to do security scanning
-2. TorchServe's Docker image is configured to expose the ports `8080`, `8081`, `8082`, `7070`, `7071` to the host by [default](https://github.com/pytorch/serve/blob/master/docker/Dockerfile). When these ports are mapped to the host, make sure to use `localhost` or a specific ip address.
+    These ports are accessible to `localhost` by default.  The addresses can be configured by following the guides for
+    [HTTP](https://github.com/pytorch/serve/blob/master/docs/configuration.md#configure-torchserve-listening-address-and-port) and
+    [gRPC](https://github.com/pytorch/serve/blob/master/docs/configuration.md#configure-torchserve-grpc-listening-addresses-and-ports).
+    TorchServe does not prevent users from configuring the address to be any value, including the wildcard address `0.0.0.0`.
+    Please be aware of the security risks of configuring the address to be `0.0.0.0`, this will give all addresses(including publicly accessible addresses, if any)
+    on the host, access to the TorchServer endpoints listening on the ports shown above.
+2. TorchServe's Docker image is configured to expose the ports `8080`, `8081`, `8082`, `7070`, `7071` to the host by [default](https://github.com/pytorch/serve/blob/master/docker/Dockerfile). When starting the container,
+   make sure to map the ports exposed by the container to `localhost` ports or a specific IP address as shown in this [security guideline](https://github.com/pytorch/serve/blob/master/docker/README.md#security-guideline).
 
 3. Be sure to validate the authenticity of the `.mar` file being used with TorchServe.
-    1. A `.mar` file being downloaded from the internet from an untrustworthy source may have malicious code, compromising the integrity of your application
-    2. TorchServe executes arbitrary python code packaged in the `mar` file. Make sure that you've either audited that the code you're using is safe and/or is from a source that you trust
-4. By default TorchServe allows you to register models from all URLs. Make sure to set `allowed_urls` parameter in config.properties to restrict this. You can find more details in the [configuration guide](https://pytorch.org/serve/configuration.html#other-properties)
-    - `use_env_allowed_urls=true` is required in config.properties to read `allowed_urls` from environment variable
+    1. A `.mar` file being downloaded from the internet from an untrustworthy source may have malicious code, compromising the integrity of your application.
+    2. TorchServe executes arbitrary python code packaged in the `mar` file. Make sure that you've either audited that the code you're using is safe and/or is from a source that you trust.
+    3. Torchserve supports custom [plugins](https://github.com/pytorch/serve/tree/master/plugins) and [handlers](https://github.com/pytorch/serve/blob/master/docs/custom_service.md).
```

Comment:
Its not clear what the suggestion is. Could you please elaborate

Reply:
The goal is to show that a custom TorchServe security scanning plugin can be created which makes use of existing tools such as [clamd](https://pypi.org/project/clamd/), [virstotal](https://virustotal.github.io/vt-py/) and [fickling](https://github.com/trailofbits/fickling). It could be an endpoint plugin which:
1. Accepts a model archive name or url as input
2. Scans the model archive contents for security issues
3. Places model archive in model store only once security scanning has passed

The above is just an example of functionality that an endpoint plugin can provide but can be customized based on the runtime security scanning requirements.